tests/lib/helper.ts

```diff
@@ -1,6 +1,12 @@
 import { expect, Page } from '@playwright/test';
 import config from '../../tailwind.config.cjs';
 
+// Convert from '<number>px' to actual number
+// ex. '3.14px' --> 3.14
+export const pxStringToNumber = (pxString: string) => {
+  return Number(pxString.substring(0, pxString.indexOf('px')));
+};
+
```

Comment:
Do we need a helper for this? I think this is the default behavior of `Number.parseFloat`

```js
> Number('3.14px')
NaN
> Number.parseFloat('3.14px')
3.14
```

Reply:
ah you are totally correct. thank you for catching!